handler/src/main/java/io/netty/handler/ssl/SslProtocols.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+/**
+ * SSL/TLS protocols
+ */
+public final class SslProtocols {
```

Comment:
I think we should add `@Deprecated` annotations on the protocols that aren't TLS v1.2 or TLS v1.3. Use of those older protocols is discouraged now a days, as far as I understand.

Reply:
@normanmaurer This looks like a very good practice. WDYT?

Reply:
Yep... Also add a `@deprecated` javadoc tag and tell why it is deprecated